healthtools_ke_api/views/sms_handler.py

```diff
@@ -233,6 +237,58 @@ def parse_elastic_search_results(response):
     return result_list
 
 
+def check_message(msg):
+    # check the message and if query wasn't understood, post error
+    if 'could not find' in msg:
+        print_error(msg)
+
+
+def print_error(message):
+    """
+    print error messages in the terminal
+    if slack webhook is set up, post the errors to slack
+    """
+    print("[{0}] - ".format(datetime.now().strftime("%Y-%m-%d %H:%M:%S")) + message)
+    response = None
+    if SLACK["url"]:
+        response = requests.post(
+            SLACK["url"],
+            data=json.dumps(
+                {
+                    "attachments":
+                        [
+                            {
+                                "author_name": "HealthTools API - Kenya",
+                                "color": "danger",
```

Comment:
@DavidLemayian that's . Slack's API. Those are the keys for formatting the message to slack.
Refer to 👉 [Webhook message formatting](https://api.slack.com/docs/message-attachments)